pandas/conftest.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+
+
+def pytest_addoption(parser):
+    parser.addoption("--skip-slow", action="store_true",
+                     help="skip slow tests")
+    parser.addoption("--skip-network", action="store_true",
+                     help="run network tests")
+    parser.addoption("--only-slow", action="store_true",
+                     help="run only slow tests")
+
+
+def pytest_runtest_setup(item):
+    if 'slow' in item.keywords and item.config.getoption("--skip-slow"):
+        pytest.skip("skipping due to --skip-slow")
+
+    if 'slow' not in item.keywords and item.config.getoption("--only-slow"):
+        pytest.skip("skipping due to --only-slow")
+
+    if 'skip' in item.keywords and item.config.getoption("--skip-network"):
```

Comment:
on the todo list, let's just make ``@network`` an actual marker then this becomes trival